web_timeline/README.rst

```diff
@@ -2,7 +2,7 @@
 Web timeline
 ============
 
-.. 
+..
```

Comment:
These changes seem to be done by the IDE stripping out leading spaces. They should be removed.